plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAdlsConnectorSmokeTest.java

```diff
@@ -88,7 +88,7 @@ protected HiveMinioDataLake createHiveMinioDataLake()
         FileAttribute<Set<PosixFilePermission>> posixFilePermissions = PosixFilePermissions.asFileAttribute(PosixFilePermissions.fromString("rw-r--r--"));
         Path hadoopCoreSiteXmlTempFile = Files.createTempFile("core-site", ".xml", posixFilePermissions);
         hadoopCoreSiteXmlTempFile.toFile().deleteOnExit();
-        Files.write(hadoopCoreSiteXmlTempFile, abfsSpecificCoreSiteXmlContent.getBytes(UTF_8));
+        Files.writeString(hadoopCoreSiteXmlTempFile, abfsSpecificCoreSiteXmlContent);
```

Comment:
There're other 2 usages of `Files.write()` in Delta Lake connector.

Reply:
The other 2 places do not have String value but have byte[] value.

https://github.com/trinodb/trino/blob/98b274bf9a19a11fff870e5dea486042435f2a7b/plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeGcsConnectorSmokeTest.java#L89-L93

https://github.com/trinodb/trino/blob/98b274bf9a19a11fff870e5dea486042435f2a7b/plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/metastore/glue/TestDeltaLakeGlueMetastore.java#L269-L270

Reply:
I didn't look into the 1st case. Regarding the 2nd case, there's `com.google.common.io.Resources#toString(URL url, Charset charset)` method.

Reply:
Thanks for the pointer. Addressed comment.